integration_tests/dbt_project.yml

```diff
@@ -1,7 +1,7 @@
 config-version: 2
 
 name: 'zendesk_integration_tests'
-version: '0.16.0'
+version: '0.17.0'
```

Comment:
Can we add the variable configuration `zendesk__unstructured_enabled: true` in the vars config here so it can be enabled when generating the docs.

Reply:
@fivetran-joemarkiewicz At the time I did this I thought we didn't want this added to the manifest, and therefore not the docs. Is this not the case?